inst/stan/base/survival.stan

```diff
@@ -57,6 +57,26 @@ functions {
         vector[rows(time)] result = - (time / 2) .* (nodes_time_hazard * weights);
         return result;
     }
+    
+    
+    // Expand design matrix by parameter vector
+    vector get_os_cov_contribution(matrix design, vector beta) {
+        int n = rows(design);
+        int p = cols(design);
+        vector[n] cov_contribution;
+        if (p == 0) {
+            // 0 Vector to cover case where no covariates are defined
+            // (e.g. intercept only model)
```

Comment:
Just to confirm: normally the design matrix we use for the covariate contribution would not contain a column of 1s, ie corresponding to an intercept?

Reply:
yeah and in the code itself do we remove the 1s column of the design matrix before passing it to Stan?

Reply:
ok all good then